src/testing/util.ts

```diff
@@ -19,9 +19,10 @@ export function render(platform: PlatformApi, html: string): Promise<HostElement
   return waitForLoad(platform, node);
 }
 
-export function transpile(tsFileName: string): string {
+export function transpile(tsFileName: string, srcDir: string): string {
   const config = mockBuildConfig();
   config._isTesting = true;
+  config.srcDir = srcDir;
```

Comment:
Transpiler was throwing rootDir error without this set properly. Filter sends in './src' (typically)